peviitor_scraper.js

```diff
@@ -57,8 +57,10 @@ function postApiPeViitor(data, company, apikey = null) {
   const scraper = new ApiScraper();
   let resolveApi = "https://dev.laurentiumarian.ro/scraper/based_scraper_js/";
   let status = { status: company.company.toLowerCase() + ".js" };
+  const axios = new ApiScraper(resolveApi);
```

Comment:
this looks a bit confusing but it should be fine

Reply:
yes i know, it's scary, right? 😂